tests/MongoDB.Driver.Core.Tests/Core/Operations/BulkMixedWriteOperationTests.cs

```diff
@@ -1117,7 +1115,7 @@ public void WriteConcern_should_work()
                 WriteConcern = WriteConcern.Unacknowledged
             };
 
-            using (var readWriteBinding = CreateReadWriteBinding())
+            using (var readWriteBinding = CreateReadWriteBinding(useExplicitSessionIfSupported: false))
```

Comment:
Without this change (and next 3), the tests failed because of a new exception for `explicit` sessions and `unacknowledged` writes